apps/vast/pyvast_threatbus/metrics.py

```diff
@@ -89,3 +89,40 @@ def reset(self):
         super(Gauge, self).reset()
         with self._lock:
             self.value = 0
+
+
+class InfiniteGauge(Gauge):
```

Comment:
Why do we inherit from `Gauge` here as opposed to `Metric`? As far as I can tell, `Gauge::value` is never used.

Reply:
Because otherwise we'd have to copy over the code for `inc()` and `dec()`. We avoid code duplication this way. The `.value` is used in the `write_metrics` function `pyvast_threatbus.py`

Reply:
Ah, right, I keep forgetting about python's hacky way of doing instance variables.